gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -253,6 +272,97 @@ def GetItemDescription (item, itemtype):
 	text = searchRE.sub(replacement, text)
 	return text
 
+def GetIWD2RestrictedAlignments(mask):
+	text = ""
+	if mask & 0x8000: # neutral
+		text += GemRB.GetString(31535) + "\n  "
+	if mask & 0x4000: # good
+		text += GemRB.GetString(31532) + "\n  "
+	if mask & 0x2000: # evil
+		text += GemRB.GetString(31537) + "\n  "
+
+	return text
+
+def GetIWD2RestrictedStances(mask):
+	text = ""
+	if mask & 0x20000: # neutral
+		text += GemRB.GetString(31535) + "\n  "
+	if mask & 0x10000: # lawful
+		text += GemRB.GetString(29372) + "\n  "
+	if mask & 0x1000: # chaotic
+		text += GemRB.GetString(31536) + "\n  "
```

Comment:
Looks like you misassigned some strings. My dump has:

> @31530 = ~Not Usable By:~
> @31531 = ~Usable By:~
> @31532 = ~Good~
> @31533 = ~Lawful~
> @31534 = ~Neutral~
> @31535 = ~Chaotic~
> @31536 = ~Evil~

Reply:
Alternatively you could just dump them into a new aligns.2da-like table for clean lookups.

Reply:
Thanks, the search of NI runs somewhat async or something, I have spent too much time on finding the strings correctly.

Reply:
re. new 2DA: is it fine to only create one of IWD2 for now? Don't want to go through this again if not required. 🥲

Reply:
Of course, we don't need it in other games.